website/docs/r/project_invitation.html.markdown

```diff
@@ -50,13 +43,7 @@ resource "mongodbatlas_project_invitation" "test" {
 
 * `project_id` - (Required) Unique 24-hexadecimal digit string that identifies the project to which you want to invite a user.
 * `username` - (Required) Email address to which Atlas sent the invitation. The user uses this email address as their Atlas username if they accept this invitation.
-* `roles` - (Required) List of Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. Atlas accepts the following roles:
-  * GROUP_OWNER
-  * GROUP_CLUSTER_MANAGER
-  * GROUP_READ_ONLY
-  * GROUP_DATA_ACCESS_ADMIN
-  * GROUP_DATA_ACCESS_READ_WRITE
-  * GROUP_DATA_ACCESS_READ_ONLY
+* `roles` - (Required) List of Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them.
```

Comment:
This needs the longer info you currently only have in the data source docs.

Reply:
A slight variation of that documentation exists on line 15 of this file as well.

Should I remove the portion that's on line 15? Keep both?

Reply:
Keep both - I think it looks good as is for now.  TY!